core/audio/start.sh

```diff
@@ -86,4 +86,9 @@ if [[ -n "$SOUND_ENABLE_SOUNDCARD_INPUT" ]]; then
   route_input_source
 fi
 
-exec pulseaudio --file /etc/pulse/balena-sound.pa
\ No newline at end of file
+exec pulseaudio \
+  --realtime=true \
+  --system=true \
+  --log-target=stderr \
+  --daemonize=false \
```

Comment:
This looks interesting too. Can you take it to the `audio` block repo [here](https://github.com/balenablocks/audio)? You can open an issue or PR over there. That repo should handle all patches/issues regarding the audio/pulseaudio stuff

Reply:
I've upstream some of the changes to the audio block repo. But FWIW, the changes will still need to be written here right? As this project is using a different `start.sh` right?

Reply:
Hi there! Changes made in the audio repo get baked into the docker image that then gets used in balenaSound. So there shoudn't be any changes needed in this repo.